File: Python/26.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        unique_idx = 1
        for org_idx in range(len(nums) - 1):
            if nums[org_idx] != nums[org_idx + 1]:
                nums[unique_idx] = nums[org_idx + 1]
                unique_idx += 1
        if len(nums) == 0:
            return 0
        return unique_idx


s1 = Solution()
s1.removeDuplicates([1, 1, 1, 2, 2, 3])

File: Java/26.java
class Solution26 {
    public int removeDuplicates(int[] nums) {
        int org_index = 0;
        int unique_index = 1;

        for (org_index = 0 ; org_index < nums.length - 1; org_index++){
            if (nums[org_index] != nums[org_index + 1]){
                nums[unique_index] = nums[org_index + 1];
                unique_index += 1;


            }

        }
        return unique_index;
    }
}